EIPS/eip-7901.md

```diff
@@ -0,0 +1,616 @@
+---
+eip: 7901
```

Comment:
```suggestion
eip: 7904
```

Don't forget to update links, rename this file, and your assets.

Reply:
done